en/05_Contributing/00_Issues_and_Bugs.md

```diff
@@ -1,96 +1,77 @@
+for you, then [submitting a pull request](/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)
+is the best way to ensure it gets fixed.
 
 ## Feature Requests
 
 [warning]
-Please don't file "feature requests" as Github issues. If there's a new feature 
-you'd like to see in SilverStripe, you either need to write it yourself (and 
-[submit a pull request](/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request) or convince somebody else to 
-write it for you. Any "wishlist" type issues without code attached can be 
+Please don't file feature requests as Github issues. If there's a new feature
+you'd like to see in Silverstripe CMS, you either need to write it yourself (and
+[submit a pull request](/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)) or convince somebody else to
+write it for you. Any "wishlist" type issues without code attached can be
 expected to be closed as soon as they're reviewed.
 [/warning]
 
-In order to gain interest and feedback in your feature, we encourage you to 
+In order to gain interest and feedback in your feature, we encourage you to
 present it to the community through the [community channels](https://www.silverstripe.org/community).
 
 ## Reporting Security Issues
 
+[warning]
 If you think a bug may have security implications, do not create a GitHub issue for it. This may lead to a zero-day vulnerability.
+[/warning]
 
-Report potential security issues to [security@silverstripe.org](mailto:security@silverstripe.org). 
-See our "[Release Process](/contributing/release_process/#security-releases)" documentation for more info, and 
-read our guide on [how to write secure code](/developer_guides/security/secure_coding/).
+Report potential security issues to [security@silverstripe.org](mailto:security@silverstripe.org). Emails sent to that address are
+forwarded to a private mailing list and kick off a specific security process.
 
-Silverstripe CMS does not operate a *bug bounty* program.
+If you have any doubts or are unsure whether the bug you've found has security implications or not, please err on the side of caution
+and email us about it.
 
-Review our [Managing Security Guidelines](managing_security_issues) guidelines to understand what happens once a vulnerability is reported.
+Review our [Managing Security Issues](managing_security_issues) process to understand what happens once a vulnerability is reported.
 
 Silverstripe CMS aims to ship security patches at pre-defined intervals when those issues are not actively exploited in the wild.
-
 Review the [Security patch windows](../Project_Governance/Minor_release_policy#security-patch-windows) section of our minor release policy for more details.
 
-## Sharing your Opinion
-
-* [forum.silverstripe.org](http://forum.silverstripe.org): Forums on silverstripe.org
-* [All issues across modules](https://www.silverstripe.org/community/contributing-to-silverstripe/github-all-core-issues)
```

Comment:
This section has no information in it... I don't know what the purpose of it was so I removed it outright.